src/types.rs

```diff
@@ -292,6 +294,31 @@ pub struct RuntimeConfig {
 	pub max_kad_record_size: u64,
 	/// The maximum number of provider records for which the local node is the provider. (default: 1024).
 	pub max_kad_provided_keys: u64,
+	/// Avail account secret key. (default: None)
+	#[serde(skip)]
```

Comment:
I think it will skip it also during deserialization: https://serde.rs/field-attrs.html#skip

Reply:
Right, fixed 👍